README.md

```diff
@@ -38,13 +35,82 @@ Notebooks supposed to be executed on https://colab.research.google.com are avail
 - [Ko_multi Step001680000 Predicted](https://soundcloud.com/user-623907374/step001680000-predicted)
 - [Ko_multi Step001700000 Predicted](https://soundcloud.com/user-623907374/step001700000-predicted)
 
+### Online TTS demo
+
+Notebooks supposed to be executed on https://colab.research.google.com are available:
+
+- [DeepVoice3: Multi-speaker text-to-speech demo](https://colab.research.google.com/github/r9y9/Colaboratory/blob/master/DeepVoice3_multi_speaker_TTS_en_demo.ipynb)
+- [DeepVoice3: Single-speaker text-to-speech demo](https://colab.research.google.com/github/r9y9/Colaboratory/blob/master/DeepVoice3_single_speaker_TTS_en_demo.ipynb)
+
+## Installation and test of version v0.1.0
```

Comment:
I don't think the script is worth putting it to the README. Setup procedure depends on envinronments where people work.

What about making your own gist for the script and put the link to the trouble shooting or a new section (e.g., Examples)? 

Also, if you like self-contained shell script that runs everything to prepare end-to-end TTS models, it might be useful to create Docerfile instead.

Reply:
I think that at least the part:
```
pip install numpy # Required because nnmnkwii setup has it as dependency and else pip install nnmnkwii (within next command) fails
pip install -e ".[bin]"
pip install matplotlib # This is not installed by previous command. Maybe it should be added to the bin target
pip install tensorflow # No special nn routines are used. A proposed patch gets rid of this requirement

python -c "import nltk; nltk.download('cmudict')"
python -c "import nltk; nltk.download('punkt')"
```
should be included.
To me at least it is confusing when I follow the requirements described in a readme, start executing the code and then get errors over and over again, having to "decipher" each one, researching what is missing, and having to install it.
So I appreciate and exhaustive requirements or install list.

The rest was information on how the models were trained and how they worked. I actually trained them with those commands.

Also, I will consider your gist suggestion, but go ahead for now merging only the additions you feel appropriate.